src/tiny-bang-ast/tiny_bang_ast.ml

```diff
@@ -149,14 +152,14 @@ type expr = Expr of ast_uid * clause list
 and clause = Clause of ast_uid * var * redex
 
 (** The type of a reducable expression. *)
-and	redex =
```

Comment:
I see you like sneaking in some code gardening on your commits. I appreciate the style :+1: